lib/diagnostic.js

```diff
@@ -38,19 +48,33 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+      name: 'Skunk',
+      species: 'dog',
+      breed: 'golden retreiver',
+      vocalization: 'woof',
+      vocalize: function(){
+        return 'my name is ' + name + ' and i say ' + vocalization;
+      }
+    };
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function() {
-  /* Your answer here */
+const Pet = function(name, species, breed, vocalization) {
+    this.name = name,
+    this.species = species,
+    this.breed = breed,
+    this.vocalization = vocalization
+    }
 };
 
 // 5. Attach a `vocalize` method to the Pet's prototype.
 
-/* your answer here */
+Pet.prototype.vocalize = function(){
+  return 'wooooooooof';
```

Comment:
As in Q3, this method should print out the value stored on a pet's `vocalize` property. Can you think of how you might do that?